doc/hardware/porting/board_porting.rst

```diff
@@ -129,6 +129,10 @@ board directory in the zephyr repository, but it's the easiest way to get
 started. See :ref:`custom_board_definition` for documentation on moving your
 board directory to a separate repository once it's working.)
 
+.. note::
+
+  The board directory name does not actually need to match the name of the board. Multiple boards can even defined be in one directory. The most common example of this is for boards with multiple targets, such as each core of a multi-core soc.
```

Comment:
target can mean a lot of things in the build system.

How about:
```suggestion
  The board directory name does not need to match the name of the board.
  Multiple boards can be defined in one directory.
  The most common example of this is boards with a muti-core SoC where each core
  is created as a logical board following the board naming scheme
   `<board>_<soc-core>`.
```

Also please pay attention to you line length.

Reply:
thank you for your suggestions